### ISSUE TITLE
Nvda stopped reading headlines named video in Yutube in microsoft edge

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7539]
Hi, I noticed today that NVDA doesn't read the name video on Yutube
.
1. Open Microsoft Edge.
2 Open www.yutube.com
3. Press H. The NVDA should read the headings with the name video and so on. Nvda says it's empty. If the NVDA says that OCR is empty, the result is the title..nvda next-14358,f53e4910.
windows 10 1703 .

**Comment by feerrenrut** [comment]
@gdata1 Could you comment on why you closed this? Is this now working for you?

**Comment by gdata1** [comment]
Hello. It doesn't work for me.

**Comment by ehollig** [comment]
Closing as duplicate of #7611, as it contains more information.
